### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_ Access spark history,Start the Amabri Portal_Keyboard: All the links present in the context menu are not displayed on the screen when user right clicks on the cluster with the help of keyboard.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3607]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingfor IntelliJ/Eclipse–New Feature; #MAS2.1.1; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS 2.1.1

**Repro Steps:**
1.Download IntelliJ application from the URL: https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA .
3.Tab till Create New Project button and press enter
4.Tab till view-->Tool Window-->Azure Explorer and press enter
5.Tab till Azure dropdown and expand it using keyboard.
6.Tab till cluster name under HD Insight and open the context menu using keyboard and select Open Spark History UI/ Open Cluster Management Portal(Ambari)
7.Verify whether the context menu is displayed using keyboard or not.

**Actual Result:**
All the links present in the context menu are not displayed on the screen when user right clicks on the cluster with the help of keyboard.

**Expected Result:**
All the links present in the context menu should be displayed on the screen when user right clicks on the cluster with the help of keyboard.

**User Impact:**
Keyboard users will face difficulty in knowing what links are present in that clusters if all links are not displayed on the screen using keyboard. 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Bug_Inconsistency is observed while opening the context menu using keyboard for start ambari portal.pptx](https://github.com/microsoft/azure-tools-for-java/files/3645469/Bug_Inconsistency.is.observed.while.opening.the.context.menu.using.keyboard.for.start.ambari.portal.pptx)

**Comment by t-rufang** [comment]
Note: No way to use keyboard to trigger the right-click menu.
![image](https://user-images.githubusercontent.com/32627233/65660191-740cf800-e060-11e9-9122-8a41d5424f7c.png)

**Comment by t-rufang** [comment]
Actually you can use the "Apps" key in the keyboard to trigger the menu on a node. Check the gif below.
![image](https://user-images.githubusercontent.com/32627233/67483923-7717fa00-f699-11e9-8430-2093ce556420.png)

![issue_3607](https://user-images.githubusercontent.com/32627233/67483885-5c458580-f699-11e9-874e-73550a367f69.gif)

**Comment by v-lasiva** [comment]
Regressed the issue in the URL mentioned in the repro steps having the details,

Version: 2019.2.2
Build: 192.6603.28

Now all the links present in the context menu are displayed on the screen when user right clicks on the cluster with the help of keyboard also.
As the issue is resolved, Hence closing the bug.
Added attachment for reference.

[Bug_3607_Issue Fixed.pptx](https://github.com/microsoft/azure-tools-for-java/files/3819251/Bug_3607_Issue.Fixed.pptx)
dded attachment for reference.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749709
